### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by wtay**
Read this before you submit this pull request:
https://github.com/mpv-player/mpv/blob/master/DOCS/contribute.md

Reading this link and following the rules will get your pull request reviewed
and merged faster. Nobody wants lazy pull requests.

### CODE REVIEW

**Comment by wtay** [comment]
I had to remove the b->requested patch, it seems to fail.

**Comment by wtay** [comment]
I added a fix for the low volume.

**Comment by pinkflames** [comment]
FWIW, I have tested these commits on Gentoo's live mpv ebuild and they are working fine. The volume fix is particularly nice.

**Comment by t-8ch** [comment]
@philipl Looks good to me.

FYI @wtay is the author of Pipewire.

File: audio/out/ao_pipewire.c [hunk 1]
```diff
@@ -187,13 +187,12 @@ static void on_state_changed(void *userdata, enum pw_stream_state old, enum pw_s
...
 {
-        return cbrt(vol) * 100;
+        return vol * 100;
 }
 
 static float mp_volume_to_spa_volume(float vol)
 {
-        vol /= 100;
-        return vol * vol * vol;
+        return vol /= 100;
```
Comment (philipl) [inline audio/out/ao_pipewire.c:195]: This can just be vol / 100. No need to assign,

**Review by philipl — COMMENTED** [review]
@wtay Just one small nit and then it'll be good to merge.

**Pull request merged**